services/greasyfork/greasyfork-license.service.js

```diff
@@ -0,0 +1,42 @@
+import { renderLicenseBadge } from '../licenses.js'
+import { InvalidResponse } from '../index.js'
+import BaseGreasyForkService from './greasyfork-base.js'
+
+class GreasyForkLicense extends BaseGreasyForkService {
+  static category = 'license'
+
+  static route = {
+    base: 'greasyfork/l',
+    pattern: ':scriptId',
+  }
+
+  static examples = [
+    {
+      title: 'Greasy Fork',
+      namedParams: { scriptId: '407466' },
+      staticPreview: this.render({ licenses: ['MIT'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  static render({ licenses }) {
+    return renderLicenseBadge({ licenses })
+  }
+
+  async handle({ scriptId }) {
+    const data = await this.fetch({ scriptId })
+    if (data.license === null) {
+      throw new InvalidResponse({
+        prettyMessage: 'version not found',
+      })
+    }
+    // remove suffix " License" from data.license
+    const license = data.license.replace(/ License$/, '')
+    return this.constructor.render({
+      licenses: [license],
+    })
```

Comment:
This is fine and don't feel obligated to make any changes, but for awareness, a common pattern we use in our service classes is to incorporate a `transform` function which takes the raw response and does any such extraction and/or transformation. We do this both for cleaner separation of concerns and because it's much easier to test with standard unit tests vs. trying to use nock with mocked requests for validating code-level statements